galata/test/jupyterlab/zoom.test.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, test } from '@jupyterlab/galata';
+
+test.describe('Low Vision / Zoom Support', () => {
+  // test('Launch Screen at 400% zoom', async ({ page }) => {
+  //   await page.evaluate('document.body.style.zoom = 4.0');
+  //   expect(await page.screenshot()).toMatchSnapshot(
+  //     'launch-screen-at-400-zoom.png'
+  //   );
+  // });
+  const { firefox } = require('playwright');
+
+  test('Should show visibility of menu bar at 400% zoom', async ({ page }) => {
+    let browser = await firefox.launch();
+    page = await browser.newPage();
+    await page.setViewportSize({ width: 1280, height: 1024 });
```

Comment:
Why set the viewport size and why this particular size?

Reply:
I have set the viewport size as this 1280px is how I've been testing the zoom function. However i'm not getting the results I need for the snapshot so will take this out of the code.